lib/exercises.js

```diff
@@ -20,10 +88,82 @@ function top_k_frequent_elements(list, k) {
 //   Each element can either be a ".", or a digit 1-9
 //   The same digit cannot appear twice or more in the same 
 //   row, column or 3x3 subgrid
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n)
+// Space Complexity: O(n)
 function valid_sudoku(table) {
```

Comment:
👍 Fun fact, since a Sudoku board is **always** 9x9, this problem is actually O(1) !